harvesters/src/main/java/org/fao/geonet/kernel/harvest/HarvestManagerImpl.java

```diff
@@ -79,6 +82,10 @@
 
     private Map<String, AbstractHarvester> hmHarvesters   = new HashMap<String, AbstractHarvester>();
     private Map<String, AbstractHarvester> hmHarvestLookup= new HashMap<String, AbstractHarvester>();
+    
+    private final List<String> summaryHarvesterSettings = Arrays.asList("harvesting","node","site","name","uuid","url","capabUrl","baseUrl","useAccount","ogctype","options","status","info","lastRun");
```

Comment:
Status and ogctype are also common, no ? And they are necessary when load list. 
Also for lastRun, url, they are used in the report in new tab. Only "capabUrl" and "baseUrl" are specifics.